mantidproject/mantid
# Fixed IDFs for both hfir instruments

Just a fix for both HFIR SANS instruments:

- None of them had a component/shape for monitors preventing from using the Solid Angle calculation in Mantid.
- GPSANS was showing weird solid angles with cylindrical pixels. They are now rectangular.

**To test:**

- Tests should pass.
- Ojne can also use `LoadEmptyInstrument` to make sure the files load and have a sensible detector shape.

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/IndividualTicketTesting/)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
